### PR TITLE
Shorten cloud comms log trace strings

### PR DESCRIPTION
**Pull request by jw-msft**
- [shorten some adps gen2 log strings](https://github.com/Azure/iot-hub-device-update/commit/5ab94306e2a3ce37e169ee4c3547ca20b6cbd3fd)
- [shorten cloud comm log trace strings](https://github.com/Azure/iot-hub-device-update/commit/c16be6cb2e3a53fdf887d0523be41319f9cc0e63)

### CODE REVIEW

File: src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c [hunk 1]
```diff
@@ -71,26 +71,26 @@ bool ReadAzureDPS2MqttSettings(AZURE_DPS_2_MQTT_SETTINGS* settings)
...
-        Log_Error("Cannot read registrationId field.");
+        Log_Error("get registrationId");
         goto done;
     }
 
     // For DPS connection, the clientId and registrationId fields are the same.
     if (mallocAndStrcpy_s(&settings->mqttSettings.clientId, settings->registrationId) != 0)
     {
-        Log_Error("Cannot allocate clientId field.");
+        Log_Error("alloc clientId");
```
Comment (JeffMill) [inline src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c:81]: not much value logging on OOM.  If we're really OOM, logging would probably fail too.

File: src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c [hunk 2]
```diff
@@ -102,14 +102,15 @@ bool ReadAzureDPS2MqttSettings(AZURE_DPS_2_MQTT_SETTINGS* settings)
 
     if (settings->mqttSettings.username == NULL)
     {
-        Log_Error("Cannot generate username field. (idScope:%s, registrationId:%s, apiVersion:%s)");
+        Log_Error("fail gen username(idScope:%s, registrationId:%s, apiVersion:%s)");
```
Comment (JeffMill) [inline src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c:105]: shouldn't there be arguments passed here?

File: src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c [hunk 3]
```diff
@@ -102,14 +102,15 @@ bool ReadAzureDPS2MqttSettings(AZURE_DPS_2_MQTT_SETTINGS* settings)
...
         goto done;
     }
 
     // NOTE: This is the 'globalDeviceEndpoint' field in the config file.
     if (!ADUC_AgentInfo_ConnectionData_GetStringField(
             agent_info, "dps.globalDeviceEndpoint", &settings->mqttSettings.hostname))
     {
+        Log_Info("default hostname: %s", DEFAULT_DPS_GLOBAL_ENDPOINT);
```
Comment (JeffMill) [inline src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c:113]: add "using"

File: src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c [hunk 4]
```diff
@@ -118,7 +119,7 @@ bool ReadAzureDPS2MqttSettings(AZURE_DPS_2_MQTT_SETTINGS* settings)
     if (!ADUC_AgentInfo_ConnectionData_GetIntegerField(
             agent_info, "dps.mqttVersion", &tmp) || tmp < MIN_DPS_MQTT_VERSION)
```
Comment (JeffMill) [inline src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c:120]: add "using"

File: src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c [hunk 5]
```diff
@@ -129,20 +130,20 @@ bool ReadAzureDPS2MqttSettings(AZURE_DPS_2_MQTT_SETTINGS* settings)
     if (!ADUC_AgentInfo_ConnectionData_GetUnsignedIntegerField(
             agent_info, "dps.tcpPort", &settings->mqttSettings.tcpPort))
     {
-        Log_Info("Using default TCP port: %d", DEFAULT_TCP_PORT);
+        Log_Info("default tcpPort: %d", DEFAULT_TCP_PORT);
```
Comment (JeffMill) [inline src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c:133]: add "using"

File: src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c [hunk 6]
```diff
@@ -129,20 +130,20 @@ bool ReadAzureDPS2MqttSettings(AZURE_DPS_2_MQTT_SETTINGS* settings)
...
         settings->mqttSettings.tcpPort = DEFAULT_TCP_PORT;
     }
 
     if (!ADUC_AgentInfo_ConnectionData_GetBooleanField(agent_info, "dps.useTLS", &settings->mqttSettings.useTLS))
     {
-        Log_Info("UseTLS: %d", DEFAULT_USE_TLS);
+        Log_Info("default useTLS: %d", DEFAULT_USE_TLS);
```
Comment (JeffMill) [inline src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c:139]: add "using"

File: src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c [hunk 7]
```diff
@@ -129,20 +130,20 @@ bool ReadAzureDPS2MqttSettings(AZURE_DPS_2_MQTT_SETTINGS* settings)
...
         settings->mqttSettings.useTLS = DEFAULT_USE_TLS;
     }
 
     tmp = -1;
     if (!ADUC_AgentInfo_ConnectionData_GetIntegerField(agent_info, "dps.qos", &tmp) || tmp < 0 || tmp > 2)
     {
-        Log_Info("QoS: %d", DEFAULT_QOS);
+        Log_Info("default qos: %d", DEFAULT_QOS);
```
Comment (JeffMill) [inline src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c:146]: add "using" (and below where applicable)

File: src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c [hunk 8]
```diff
@@ -118,7 +119,7 @@ bool ReadAzureDPS2MqttSettings(AZURE_DPS_2_MQTT_SETTINGS* settings)
     if (!ADUC_AgentInfo_ConnectionData_GetIntegerField(
```
Comment (JeffMill) [inline src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c:119]: future:

`bool GetUIntField(obj, field, min, min, default, &output)`

would simply this code section, and can be used for "GetBooleanField" as well.

File: src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c [hunk 9]
```diff
@@ -153,14 +154,14 @@ bool ReadAzureDPS2MqttSettings(AZURE_DPS_2_MQTT_SETTINGS* settings)
     if (!ADUC_AgentInfo_ConnectionData_GetBooleanField(
             agent_info, "dps.cleanSession", &settings->mqttSettings.cleanSession))
     {
-        Log_Info("CleanSession: %d", DEFAULT_ADPS_CLEAN_SESSION);
+        Log_Info("default CleanSession: %d", DEFAULT_ADPS_CLEAN_SESSION);
         settings->mqttSettings.cleanSession = DEFAULT_ADPS_CLEAN_SESSION;
     }
 
     if (!ADUC_AgentInfo_ConnectionData_GetUnsignedIntegerField(
```
Comment (JeffMill) [inline src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c:161]: Looks like there **is** a GetUnsigned.  Places above where tmp<0 is being checked should instead use GetUnsigned, no?

**Review by JeffMill — CHANGES_REQUESTED** [review]

File: src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c (hunk 9 above)
Comment (jw-msft) [reply src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c:161]: Tracking this for later. It seems logical but the DPS is working well and don't have time right now to investigate if the dps connection data can return a negative integer for that.

File: src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c (hunk 8 above)
Comment (jw-msft) [reply src/extensions/agent_modules/cloud_communication/adps2_mqtt_client/src/adps_gen2.c:118]: tracking for future

**Pull request merged**